src/main/java/com/example/java/maven/cardGamePeterPan/Player.java

```diff
@@ -0,0 +1,86 @@
+package com.example.java.maven.cardGamePeterPan;
+
+import java.util.ArrayList;
+
+public class Player {
+    private final String name;
+    private final ArrayList<Card>hand;
+    private boolean isPlaying;
+
+    public Player(String name) {
+        this.name = name;
+        this.hand = new ArrayList<>();
+        this.isPlaying = true;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public ArrayList<Card> getHand() {
+        return hand;
+    }
+
+    @Override
+    public String toString() {
+        return name + ", with hand " + hand;
+    }
+
+    public void addCardToHand(Card card) {
+        hand.add(card);
+    }
+
+    public void discardCardsFromHand(int firstCardIndex, int secondCardIndex) {
+        hand.remove(secondCardIndex);
+        hand.remove(firstCardIndex);
+    }
+
+    public boolean areCardsMatchingBySuit(int firstCardIndex, int secondCardIndex) {
```

Comment:
ALSO :D as you can see are**Cards**Matching - so I will suggest move this method to `Card` class - so you will have something like:
```java
public boolean isMatching(Card card){
   return this.suit.equals(card.suit) || this.rank.equals(card.rank);
}

```